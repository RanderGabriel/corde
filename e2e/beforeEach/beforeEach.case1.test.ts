import { runCLI } from "../cliRunner";
import Utils from "../testUtils";

it("should print on console", async () => {
  const command = Utils.buildCommandWithConfigPath("beforeEach", "bot_case1.test.ts");
  const results = await cli.exec(command);
  expect(results.stdout).toContain("test beforeEach");
  expect(results.statusCode).toEqual(0);
});
