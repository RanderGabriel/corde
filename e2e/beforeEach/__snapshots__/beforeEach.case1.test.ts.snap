// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should print on console 1`] = `
"test beforeEach

 PASS   e2e\\\\beforeEach\\\\__cordeTest__\\\\bot_case1.test.ts   100ms
   âœ”  <empty test name>   100ms

Test Files: 1 passed. 1 total
Tests:      1 passed. 1 total
Time:       100ms
"
`;
